src/main/resources/rhizome.yaml

```diff
@@ -14,15 +14,6 @@ hazelcast:
     seed-nodes: 
         - "localhost"
 
-hazelcast-clients:
```

Comment:
Is this intentional or because of merge?

Reply:
definitely because of merge :) thanks!

Reply:
np :)